test/cql-pytest/test_uda.py

```diff
@@ -86,4 +86,32 @@ def test_incorrect_state_func(scylla_only, cql, test_keyspace):
     with new_function(cql, test_keyspace, avg2_partial_body) as avg2_partial, new_function(cql, test_keyspace, div_body) as div_fun:
         custom_avg_body = f"(bigint) SFUNC {avg2_partial} STYPE tuple<bigint, bigint> FINALFUNC {div_fun} INITCOND (0,0)"
         with pytest.raises(InvalidRequest, match="State function not found"):
-            cql.execute(f"CREATE AGGREGATE {test_keyspace}.{unique_name()} {custom_avg_body}")
\ No newline at end of file
+            cql.execute(f"CREATE AGGREGATE {test_keyspace}.{unique_name()} {custom_avg_body}")
+
+# Test that UDA works without final function and returns accumulator then
+def test_no_final_func(scylla_only, cql, test_keyspace):
```

Comment:
No need to do this right now, but it would be nice if we could find a way to write UDA tests that would work on both Scylla and Cassandra - e.g., using a different language for the function but the rest of the test is the same. Especially for this issue - where the whole point of this change was that we believed that Cassandra allows final func and init value to be missing - and it makes sense to really check this on Cassandra as well.